src/main/java/Attendance.java

```diff
@@ -0,0 +1,18 @@
+import java.util.ArrayList;
+
+public class Attendance {
+		ArrayList<Student> students = new ArrayList<Student>();
+		Course course;
+		Teacher teacher;
+		boolean present;
+
+		public Attendance(ArrayList<Student> student, Course course, Teacher teacher, boolean present) {
```

Comment:
Name of the variable pointing to an ArrayList can be considered as plural like students